integration/getmore_compat_test.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo/options"
+
+	"github.com/FerretDB/FerretDB/integration/setup"
+	"github.com/FerretDB/FerretDB/integration/shareddata"
+)
+
+type queryGetMoreCompatTestCase struct {
+	sort      bson.D
+	batchSize int
+	limit     int
+}
+
+func testGetMoreCompat(t *testing.T, testCases map[string]queryGetMoreCompatTestCase) {
```

Comment:
I'm not seeing where it is calling getMore :thinking:

Reply:
It's called by the driver when there is more document than in one batch. Added a comment about it where the `All` method is called.